website/screens/components/select/code/examples/basic.ts

```diff
@@ -0,0 +1,31 @@
+import { DxcSelect, DxcStack, DxcInset } from "@dxc-technology/halstack-react";
+import { useState } from "react";
+
+const code = `() => {
+  const options = [
+    { label: "Madrid", value: "madrid" },
+    { label: "Melbourne", value: "melbourne" },
+    { label: "London", value: "london" },
+    { label: "Roma", value: "roma" },
+  ];
+
+  return (
+    <DxcInset space="large">
+      <DxcSelect
+        label="Select your favorite city"
+        placeholder="Choose a city"
+        options={options}
+        size="fillParent"
```

Comment:
Remove size to make code cleaner.

Reply:
We finally decided to stay with `fillParent` so I close these comments.